zerver/webhooks/freshping/tests.py

```diff
@@ -0,0 +1,34 @@
+from zerver.lib.test_classes import WebhookTestCase
+
+
+class FreshpingHookTests(WebhookTestCase):
+    STREAM_NAME = "freshping"
+    URL_TEMPLATE = "/api/v1/external/freshping?api_key={api_key}&stream={stream}"
+    FIXTURE_DIR_NAME = "freshping"
+
+    def test_freshping_check_test(self) -> None:
+        """
+        Tests if freshping check test is handled correctly
+        """
+        expected_topic = "Freshping"
+        expected_message = "Your Freshping test webhook has been received."
```

Comment:
Generally we like these to be more like "The {name} webhook is successfully configured! :tada:".

We actually have a helper for this, `get_setup_webhook_message`, which is oddly defined in `zerver/lib/webhooks/git.py`; we should move it to `zerver/lib/webhooks/common.py` and document it as well.